scripts/mysql/upgrade-0.5.0-to-0.5.1-mysql.sql

```diff
@@ -0,0 +1,12 @@
+--
+-- Copyright 2024 Datastrato Pvt Ltd.
+-- This software is licensed under the Apache License version 2.
+--
+
+ALTER TABLE `role_meta`
+    ADD COLUMN `securable_objects` VARCHAR(2048) NOT NULL COMMENT 'securable objects' AFTER `properties`;
```

Comment:
we need an update script to migrate the existing securable_object data to the new securable_objects column

Reply:
Feature isn't available now. We don't need to assume that we have already data.

Reply:
I added the scripts.